README.md

```diff
@@ -221,3 +227,30 @@ Not all steps need to be followed on the instructions for the aad-pod-identity p
     ```
 
 **NOTE** When using the `Pod Identity` option mode, there can be some amount of delay in obtaining the objects from keyvault. During the pod creation time, in this particular mode `aad-pod-identity` will need to create the `AzureAssignedIdentity` for the pod based on the `AzureIdentity` and `AzureIdentityBinding`, retrieve token for keyvault. This process can take time to complete and it's possible for the pod volume mount to fail during this time. When the volume mount fails, kubelet will keep retrying until it succeeds. So the volume mount will eventually succeed after the whole process for retrieving the token is complete.
+
+## Contributing
+This project welcomes contributions and suggestions.  Most contributions require you to agree to a
+Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us
+the rights to use your contribution. For details, visit https://cla.opensource.microsoft.com.
+
+When you submit a pull request, a CLA bot will automatically determine whether you need to provide
+a CLA and decorate the PR appropriately (e.g., status check, comment). Simply follow the instructions
+provided by the bot. You will only need to do this once across all repos using our CLA.
+
+This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/).
+For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or
+contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
+
+
+### For learning how to contribute, please read the [Contributing Documentation](/docs/contributing.md)
+### Prerequisites
+1. Visual Studio Code with `Remote - Container` extension enabled.
+2. Azure account and subscription
+3. **optional** Azure Key Vault configured
+
+### Debugging
+
+Inside the Contributing documentation, there will be a summary on the debugging method used for the `Secrest Store CSI Driver Provider for Azure`.
+
+You can directly access the Debugging Documentation [here](/docs/debug.md#debug-guide)
+
```

Comment:
Replace all of this with a simple sentence and link to the docs/contributing.md file